pantheon/src/main/java/tech/pegasys/pantheon/RunnerBuilder.java

```diff
@@ -141,7 +141,7 @@ public RunnerBuilder webSocketConfiguration(final WebSocketConfiguration webSock
   }
 
   public RunnerBuilder permissioningConfiguration(
-      final PermissioningConfiguration permissioningConfiguration) {
+      final Optional<PermissioningConfiguration> permissioningConfiguration) {
```

Comment:
as per earlier comments, make this take a PermissioningConfiguration rather than an Optional